addons/hr_holidays/report/hr_leave_report_calendar.py

```diff
@@ -48,19 +48,21 @@ def init(self):
             em.job_id AS job_id,
             CASE
                 WHEN hl.holiday_type = 'employee' THEN rr.tz
-                ELSE %s
+                ELSE COALESCE(rc.tz, 'UTC')
             END AS tz,
```

Comment:
It's more complex than that. To match the [`_get_tz` function](https://github.com/odoo/odoo/blob/4fed190a2658008d78b0ae03ae18f6733689cca9/addons/hr/models/hr_employee.py#L395-L402), you should also get the TZ from the company's calendar

```suggestion
            COALESCE(
              CASE WHEN hl.holiday_type = 'employee' THEN COALESCE(rr.tz, rc.tz) END,
              cc.tz,
              'UTC'
            )  AS tz,
```

Where `cc` is the company's calendar
```sql
  LEFT JOIN res_company co ON co.id = em.company_id
  LEFT JOIN resource_calendar cc ON cc.id = co.resource_calendar_id
```

Reply:
Hello @KangOl,

I have just updated the patch as per your suggestion. Let me know if anything is still left in this patch.

Thanks.